src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -91,6 +95,60 @@
   [:hr (use-style separator-style)])
 
 
+(defn help-modal
+  [open?]
+  (let [close-modal         #(reset! open? false)]
+    (fn []
+      [:div (use-style modal-style)
+       [modal/modal
+
+        {:title    [:div.modal__title
+                    [:> MergeType]
+                    [:h4 "Help Popup"]
+                    [button {:on-click close-modal}
+                     [:> Close]]]
+
+        ;;  :content  [:div (use-style modal-contents-style)
```

Comment:
should this be removed?

Reply:
This will be needed in order to close the modal once it opens, but probably it should not be here. In other words, the answer to your question is yes.